apps/app-article/src/App.js

```diff
@@ -463,8 +469,7 @@ class App extends Component {
                                 {
                                     this.state.appearLogin ?
                                         <Login onRegister={() => this.onRegisterOpen()}
-                                               onUserFetchSuccess={(user) => this.onUserFetchSuccess(user)}
-                                               disableSocialLogins={[]}/>
```

Comment:
Let's leave this `disableSocialLogins` in, as a reminder for the future (we did disable some things in the last days)